ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ebgp/peer-groups.j2

```diff
@@ -24,3 +24,18 @@
       send_community: all
       maximum_routes: 0
 {% endif %}
+{% if switch.mpls_vpn_gateway is arista.avd.defined(true) %}
+    {{ switch.bgp_peer_groups.mpls_vpn_gateway_overlay_peers.name }}:
+      type: mpls-vpn
+      update_source: {{ switch.mpls_vpn_gateway_mpls_tunnel_source }}
+      bfd: true
+      ebgp_multihop: "{{ mpls_vpn_gateway_ebgp_multihop }}"
+{%     if switch.bgp_peer_groups.mpls_vpn_gateway_overlay_peers.password is arista.avd.defined %}
+      password: {{ switch.bgp_peer_groups.mpls_vpn_gateway_overlay_peers.password }}
+{%     endif %}
+      send_community: all
+      maximum_routes: {{ switch.mpls_vpn_gateway_maximum_routes }}
+{%     if switch.mpls_vpn_gateway_local_as is arista.avd.defined %}
```

Comment:
What happens here if you don't define it - does it end up being `False`  (which should be changed according to previous comments) ? However probably worth adding a test where it is not defined to make sure this is rendered correctly